packages/databricks-vscode/src/configuration/models/ConfigModel.ts

```diff
@@ -214,12 +215,32 @@ export class ConfigModel implements Disposable {
             // We want to wait for all the configs to be loaded before we emit any change events from the
             // configStateCache.
             await this.readStateMutex.synchronise(async () => {
-                await Promise.all([
+                await Promise.allSettled([
```

Comment:
Is it intentional that we never fail here anymore and always proceed to the `setAuthProvider`? `setAuthProvider` doesn't fail too with `allSettled`, so we are swallowing non-ProcessError errors now, breaking some of the higher-level error handling (in the ConnectionManager and ConnectionCommands)

Reply:
Yes.  The idea is to not block login based on errors here, and also actually run all the promises parallely. I agree that we should show error messages for other errors as well. 

Right now, when any of the models fail to refresh, we completely logout, and user has to log back in. With this change, the user can make changes and fixes to their bundle, which will automatically refresh the models.